MovingBlocks/Terasology
# Work on render nodes: Blur passes, tone mapping and final post-processing nodes

### Contains
- refactoring/cleanup/documentation for ToneMappingNode
- split BlurPassesNode into two instance of LateBlurNode
- refactoring/documentation for FinalPostProcessingNode

### How to test

- Run terasology
- Create a new game or open an existing one
- Verify there are no changes to the rendering
